torchdata/datapipes/iter/util/jsonparser.py

```diff
@@ -33,15 +33,13 @@ class JsonParserIterDataPipe(IterDataPipe[Tuple[str, Dict]]):
         [('1.json', ['foo', {'bar': ['baz', None, 1.0, 2]}]), ('2.json', {'__complex__': True, 'real': 1, 'imag': 2})]
     """
```

Comment:
Adding usage example for `input_col` argument

```suggestion
        [('1.json', ['foo', {'bar': ['baz', None, 1.0, 2]}]), ('2.json', {'__complex__': True, 'real': 1, 'imag': 2})]
    >>> datapipe_extra_col = datapipe1.map(lambda x: (x[0], "extra", x[1]))
    >>> # Use `input_col` if the stream is not in position 1
    >>> json_dp2 = datapipe_extra_col.parse_json_files(input_col=2)
    >>> list(json_dp2)
    [
        ("1.json", "extra", ["foo", {"bar": ["baz", None, 1.0, 2]}]),
        ("2.json", "extra", {"__complex__": True, "real": 1, "imag": 2}),
    ]
    """
```